src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.person-info {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.pers-inf:last-child {
```

Comment:
you don't have this class

Reply:
Remove these styles, cause you don't have this class.